components/suggestion/suggestion_list.jsx

```diff
@@ -209,12 +211,23 @@ export default class SuggestionList extends React.PureComponent {
 
         const mainClass = 'suggestion-list suggestion-list--' + this.props.location;
         const contentClass = 'suggestion-list__content suggestion-list__content--' + this.props.location;
+        let suggestionListMaxHeight = '';
+
+        suggestionListMaxHeight = windowHeight() - (this.props.wrapperRef.clientHeight + Constants.PREVIEWER_HEIGHT);
```

Comment:
careful: this.props.wrapperRef is not required, so it might not exist, what should we do if it is undefined?